### ISSUE TITLE
Get-ADServiceAccount Example 3 should be removed

### ISSUE THREAD

**Issue by mwtrigg** [MicrosoftDocs/windows-powershell-docs#2259]
Example 3 should be removed from the documentation; the cmdlet does not necessarily perform the way described in the example.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe3affb4-2f61-05e7-c600-760c25e8d066
* Version Independent ID: b02f24ae-c006-8130-42d4-89fc746a75fc
* Content: [Get-ADServiceAccount (ActiveDirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/get-adserviceaccount?view=winserver2012-ps)
* Content Source: [docset/winserver2012-ps/activedirectory/Get-ADServiceAccount.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2012-ps/activedirectory/Get-ADServiceAccount.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @kenwith
* Microsoft Alias: **kenwith**

**Comment by velkovb** [comment]
@mwtrigg 

Thank you for you feedback.

I have just tested the example in my lab and it seems valid. Here is the output that I get:
```powershell
PS C:\> Get-ADServiceAccount -Filter "HostComputers -eq 'CN=TEST2016-1,CN=Computers,DC=home,DC=lab'"  -Properties HostComputers
```
```output
DistinguishedName : CN=gmsa-Test01,CN=Managed Service Accounts,DC=home,DC=lab
Enabled           : True
HostComputers     : {CN=TEST2016-1,CN=Computers,DC=home,DC=lab}
Name              : gmsa-Test01
ObjectClass       : msDS-GroupManagedServiceAccount
ObjectGUID        : c15a47dc-aa7c-41bd-8547-c20dc0e8c820
SamAccountName    : gmsa-Test01$
SID               : S-1-5-21-2222592190-2589174168-964995591-1112
UserPrincipalName :
```

The service account has to be added with the **Add-ADComputerServiceAccount** before that.

**Comment by mwtrigg** [comment]
The example I am asking to be removed says that this will list the service accounts **installed** on the computer.

The documentation for Add-ADComputerService account specifically states this performs a different function from installing a service account.

I can get this cmd let to retrieve a list of computer object allowed to retrieve the gMSA's password _without_ using the Add-ADComputerServiceAccount (though, it doesn't work when a security group is delegated the right to retrieve a gMSA's password -- despite the fact that it is best practice).

In short, this cmdlet will not retrieve a list of computers where the account is installed - only a list of computers with sufficient access to do it.

**Comment by velkovb** [comment]
@mwtrigg Thank you for the detailed explanation. I guess I did not read through carefully.

Do you think we can just update the example description to match what you are saying instead of removing it?

I think that it also matters if we are talking about gMSA's or just regular MSA.
